src/poliastro/twobody/orbit.py

```diff
@@ -162,7 +168,19 @@ def from_body_ephem(cls, body, epoch=None):
             warn("Input time was converted to scale='tdb' with value "
                  "{}. Use Time(..., scale='tdb') instead."
                  .format(epoch.tdb.value), TimeScaleWarning)
+        if body == Moon:
+            solar_system_ephemeris.set("jpl")
```

Comment:
We should don't do this: it affects the whole program without noticing, as you saw by the test failures.

Reply:
Yeah Sure!